checks/cs50/2017/x/speller/checks.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Speller(Checks):
+
+    @check()
+    def exists(self):
+        """dictionary.c, dictionary.h, and Makefile exist"""
+        super(Speller, self).exists("dictionary.c")
+        super(Speller, self).exists("dictionary.h")
+        super(Speller, self).exists("Makefile")
+
+    @check("exists")
+    def compiles(self):
+        """speller compiles"""
+        self.include("speller.c")
```

Comment:
So I'll admit each time I read `include` I have to think about the semantics, and I worry TFs might think this means to import its contents into the problem's own code (a la our append trick). Would `install` be even clearer?